README.md

```diff
@@ -23,7 +23,11 @@ make install
 
 Now you are able to start Habitat. To run the node, run one of the following:
 ```
-make run-dev          # Runs the habitat node in dev mode
+make run-dev-compose          # Runs the habitat node in dev mode
+```
+To get added to the Tailscale tailnet the first time you run Habitat in dev mode, get an auth key issued from Tailscale and run. You only need to do this once, unless you remove the volumes in `.habitat`.
```

Comment:
How do I get this auth key in Tailscale?

Reply:
Discussed offline -- need to generate one from tailscale account or logged-in CLI. I'm setting up a 1Password to manage stuff like this.